uc/uc.py

```diff
@@ -20,6 +20,7 @@ def marshalled(self, context):
         company_report = body[0]
         company_report.set('ns1:product', self.product_code)
         company_report.set('ns1:version', '2.1')
+        print(context.envelope.getChild('Body'))
```

Comment:
@flaviamissi that is correct you need a console handler configured in your settings file (http://stackoverflow.com/questions/18920428/django-logging-on-heroku?rq=1). We already have it. What you need here is something very similar to this example: https://docs.djangoproject.com/en/1.7/topics/logging/#using-logging